### PR TITLE
[UI] renamed public alpha to a dynamic version number on the Store

### PR DESCRIPTION
**Pull request by biliesilva**
<--- Description --->
This is a small update on our version badge on the Store, changed from Alpha to a dynamic version number, and also removed the version number on the settings page as we will have this in a more prominent place now. 

---
## Before
image

## After
photo_2024-12-10_13-07-13

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by flavioislima — APPROVED** [review]
LGTM
Left a minor comment.

File: src/frontend/components/UI/AppVersion/index.tsx [hunk 1]
```diff
@@ -18,15 +16,5 @@ export default function AppVersion() {
...
-      className="body-sm"
-      style={{
-        color: 'var(--color-neutral-400)',
-        margin: 'var(--space-xl) 0px var(--space-md) 0px'
-      }}
-    >
-      {t('info.hyperplay.version', 'HyperPlay Version') + `: ${appVersion}`}
-    </div>
-  )
+  return <div>{appVersion}</div>
```
[the remark this answers is not in the document]
Comment (biliesilva) [reply src/frontend/components/UI/AppVersion/index.tsx:19]: I tried to use de "V." but as the version number is located right after the HP logo, it seems clear that is the version number, and also it's more "clean".

**Review by jiyuu-jin — APPROVED** [review]
Looks good!

**Review by eliobricenov — APPROVED** [review]

**Review by BrettCleary — APPROVED** [review]
LGTM

File: src/frontend/components/UI/AppVersion/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (flavioislima) [reply src/frontend/components/UI/AppVersion/index.tsx:19]: ok, sounds good then.

**Review by nyghtstalker — APPROVED** [review]
- Love seeing the alpha removed (props to us!!!) and having the version displayed there! Very easy to read and determine the current client version!
- Confirmed that the client version is no longer displayed in the Settings -> General tab.

image
image

**Pull request merged**